projects/or_scs/src/Mirza/SupplyChain/Handlers/UXUtils.hs

```diff
@@ -49,16 +48,24 @@ data PrettyEventResponse =
 instance FromJSON PrettyEventResponse where
   parseJSON = withObject "PrettyEventResponse" $ \v -> PrettyEventResponse
     <$> v .: "event"
-    <*> v .: "orglocation"
+    <*> v .: "location"
 
 instance ToJSON PrettyEventResponse where
-  toJSON (PrettyEventResponse ev (Just orgLoc)) = object
-    [ "eventType" .= Ev._etype ev
-    , "orgName" .= (orgResponseName . orgResponse) orgLoc
-    , "orgLocation" .= locationResponse orgLoc
+  toJSON (PrettyEventResponse (Ev.Event eType _meId dwhat dwhen (DWhy bizStep disp) _dwhere) (Just orgLoc)) = object
```

Comment:
Is this duplicating the ToJSON logic from GS1Combinators?

Can we instead call toJSON on the event, and then add an extra fields, `location` to the result?

Reply:
It is not duplicating the logic. It is sending relevant bits of the event (e.g, disposition, action) along with the location information. Although, calling toJSON on the event and appending the `location` to it would be a superset of my current `ToJSON PrettyEventResponse`